datadog_checks_dev/datadog_checks/dev/plugin/pytest.py

```diff
@@ -180,6 +182,35 @@ def run_check(config=None, **kwargs):
         yield run_check
 
 
+@pytest.fixture
+def dd_run_check():
+    def run_check(check, extract_message=False):
+        error = check.run()
+
+        if error:
+            error = json.loads(error)[0]
+            if extract_message:
+                raise Exception(error['message'])
+            else:
+                exc_lines = ['']
+                exc_lines.extend(error['traceback'].splitlines())
+                raise Exception('\n'.join(exc_lines))
+
+        return ''
+
+    return run_check
+
+
+@pytest.fixture
+def dd_get_state(key):
+    return deserialize_data(get_env_vars().get(key.lower()))
+
+
+@pytest.fixture
```

Comment:
This is a fixture with arguments, I'm not sure if and how you can use it. Can you add a working example with this?

Reply:
Oops, fixed!